sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/MessageLockToken.java

```diff
@@ -5,26 +5,28 @@
 
 import java.util.UUID;
 
+import com.azure.messaging.servicebus.models.ReceiveMode;
 /**
- * Type representing the lock-token.
+ * Type representing the lock-token. The lock token is a reference to the lock that is being held by the Service Bus in
+ * {@link ReceiveMode#PEEK_LOCK} mode.
  */
 public interface MessageLockToken {
 
     /**
      * Gets the lock token.
      *
```

Comment:
The fact that it is a UUID is an implementation detail. External customers wouldn't care; they want to know that if they pass this string into our functions, it'll do the operation for that message. Remove it from the comments.